raft/omdao_raft.py

```diff
@@ -92,7 +88,6 @@ def setup(self):
 
             mnpts = member_npts[i - 1]
             mnpts_lfill = member_npts_lfill[i - 1]
-            mnpts_rho_fill = member_npts_rho_fill[i - 1]
```

Comment:
For consistency all mnpts_rho_fill should probably be switched with mnpts_lfill and then this variable removed entirely. But I can do this in a later commit as removing the option will mean changing the tests as well.